obal/data/roles/fetch_sources/tasks/main.yml

```diff
@@ -112,22 +117,12 @@
         - '%global _release'
         - '%global release'
 
-    - name: 'add changelog entry'
-      lineinfile:
-        path: "{{ spec_file_path }}"
-        line: |
-          * {{ lookup('pipe', 'date +"%a %b %d %Y"') }} {{ downstream_changelog_name | default('Satellite6 Jenkins') }} {{ downstream_version }}-{{ downstream_release }}
-          {{ downstream_changelog }}
-        insertafter: '^%changelog'
+    - name: 'ensure changelog'
```

Comment:
`ensure changelog` sounds weird to me. the `changelog` module always (well, as long there is a `%changelog` section) adds the entry, so `add changelog entry` as the name here would be more appropriate?